tests/compiler/test_warnings.py

```diff
@@ -284,13 +294,13 @@ def test_deprecation_minus_in_typedef_default_name(snippetcompiler):
             f"(reported in Corsa-opel ({snippetcompiler.project_dir}/main.cf:6:9))"
         )
         compiler.do_compile()
-        assert len(w) == 2
-        assert issubclass(w[0].category, CompilerDeprecationWarning)
-        assert str(w[0].message) == message
+
+        assert len(caught_warnings) >= 2
```

Comment:
Do you know what it's `>=2` and not `>=1`?

Reply:
The other warning raised is this one:
`Default constructors are deprecated. Use inheritance instead. (reported in typedef Corsa-opel as Car(brand='opel') (/tmp/tmpf0yq0x__/main.cf:6))`
And I think it is already checked in this test case: https://github.com/inmanta/inmanta-core/blob/448ead859994045e27ad6120513a7042f5f60a21/tests/compiler/test_warnings.py#L162

Reply:
In that case I would change this to `>=1`.